interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -426,12 +435,19 @@ namespace {
         }
       }
 #else // _MSC_VER
+      const char* RootInclude = std::getenv("ROOT_INCLUDE");
```

Comment:
We try to layer ROOT on top of cling, as cleanly as possible. Thus cling should use ROOT_INCLUDE. Can we maybe adjust the interpreter construction instead, passing `-isystem` and `-nostdinc++`? Then cling could skip the stdinc++ determination, reacting on `-nostdinc++` being passed as an argument.